### DISCUSSION TITLE
upload csv [Apr 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1849] [Discord Threads]
> 🙋‍♂️ **mp1234**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### More -> Upload CSV  
![image0](https://cdn.discordapp.com/attachments/964457644498755644/964458740118089758/unknown.png)  
#  
> 💁 **o1lab**  
#### upload is in alpha  
#  
> 🙋‍♂️ **mp1234**  
#### yup, we are testing it at the moment  
#  
> 🙋‍♂️ **mp1234**  
#### What about appending an existing table with a csv file?  
#  
> 👨‍🚀 **wingkwong**  
#### Upload CSV = append records to a table  
#  
> 👨‍🚀 **wingkwong**  
#### u need to have an existing table first before using "Upload CSV"  
#  
> 🙋‍♂️ **Priyanka**  
#### Is there any limit of the records for updating. If yes then what is it?  
#  
> 👨‍🚀 **wingkwong**  
#### how many records you wanna import?  
#  
> 🙋‍♂️ **Priyanka**  
#### 1-2 lakh rows  
#  
> 👨‍🚀 **wingkwong**  
#### it should be fine  
#  
> 💁 **o1lab**  
#### if its a large number of records   
#### better to do it via mysql workbench or similar tools   
#### you need to figure which table is it in database and thats about it  
#  
> 🙋‍♂️ **Priyanka**  
#### ok thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/964457644498755644)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
